lib/expeditor/command.rb

```diff
@@ -38,8 +38,9 @@ def start(current_thread: false)
     # Equivalent to retryable gem options
     def start_with_retry(retryable_options = {})
       unless started?
+        current_thread = retryable_options.delete(:current_thread)
```

Comment:
First time, I tried implementing `current_thread` option as a keyword argument. It's like `def start_with_retry(retryable_options = {}, current_thread: false)`.
But it doesn't work. If it uses the implementation, we can't send retryable_options(e.g. `tries: 3`).

For example

```ruby
def start_with_retry(retryable_options = {}, current_thread: false)
  # ...
end

start_with_retry(tries: 3, sleep: 0) # unknown keywords: tries, sleep (ArgumentError)
```


So, I've implemented the options as one argument.